hardhat.config.ts

```diff
@@ -126,6 +135,14 @@ const config: HardhatUserConfig = {
           apiURL: 'https://api-sepolia-optimistic.etherscan.io/api',
           browserURL: 'https://sepolia-optimism.etherscan.io'
         }
+      },
+      {
+        network: 'zksyncSepolia',
+        chainId: 300,
+        urls: {
+          apiURL: 'https://block-explorer-api.sepolia.zksync.dev/api',
+          browserURL: 'https://sepolia.explorer.zksync.io'
+        }
```

Comment:
Can you once check if hardhat supports zksync and zksync sepolia by default or not

Reply:
Looks like there is support for zkSync deployments and verifications on hardhat as per here:

https://era.zksync.io/docs/#configuration

Reply:
https://github.com/NomicFoundation/hardhat/blob/main/packages/hardhat-verify/src/internal/chain-config.ts

I was talking about this file. It seems like even zksync mainnet network is not present in the built in chains. So that also will have to be added to customChains

Reply:
Added zkSync mainnet to hardhat config as well.